### DISCUSSION TITLE
uhhh did lax screw over EFI?

### DISCUSSION THREAD

**Discussion by STPv22** [eaglerforge/EaglerForge#50] [General]
![image](https://github.com/user-attachments/assets/db595114-e5a5-45bc-9afd-8110f0b8e969)

**Comment by STPv22** [comment]
net.minecraft is no longer in main/java?

**Comment by Project516** [comment]
shet!

**Reply by STPv22** [reply]
shittake mushrooms

**Reply by ZXMushroom63** [reply]
in u37? u37 works

**Reply by STPv22** [reply]
really? but he moved Minecraft into game/java, that doesn't mess anthing up?

**Reply by ZXMushroom63** [reply]
> really? but he moved Minecraft into game/java, that doesn't mess anthing up?

nope.

**Reply by STPv22** [reply]
whelp. imma learn more java
